server/service/src/sync/program_schemas/hiv_care_encounter_config.json

```diff
@@ -20,7 +20,7 @@
       "config": {
         "datetimeField": "arvMedication.endOfSupply",
         "scheduleFromNow": true,
```

Comment:
"scheduledFromNow" should be removed as well because it should be scheduled from the "datetimeField"... and you could remove the `"minutes": 0` entry as well